app/models/app_config.rb

```diff
@@ -95,7 +95,7 @@ def further_setup_by_category?
     if integrations.monitoring.present?
       categories[:monitoring] = {
         further_setup: integrations.monitoring.any?(&:further_setup?),
-        ready: bugsnag_ready?
+        ready: bugsnag_ready? || firebase_ready?
```

Comment:
See earlier comment, I think `firebase_crashlytics_ready?` should be a separate check.